test-resources/test/test-util/destination-service-mocks.ts

```diff
@@ -58,20 +59,168 @@ export function mockSubaccountDestinationsCall(
     .reply(responseCode, response);
 }
 
-export function mockSingleDestinationCall(
-  nockRef: nockFunction,
-  response: any,
-  responseCode: number,
-  destName: string,
-  headers: Record<string, any>,
-  options?: { uri?: string; badheaders?: string[] }
+type MockedDestinationConfiguration =
+  | (DestinationJson & {
```

Comment:
this leading `|` tripped me. I double checked it in the typescript playground, which doesn't complain about it, so I guess it's fine. wasn't aware this syntax was possible.

Reply:
This is done automatically by the linter. Initially I also stumbled over that, but when the types are longer, it is easier to grasp this way.